packages/theme-chalk/src/cascader.scss

```
@@ -157,6 +157,16 @@
     &.#{$namespace}-popper {
       box-shadow: getCssVar('cascader-menu-shadow');
     }
+
+    .#{$namespace}-cascader-panel {
+      overflow-x: hidden;
+    }
+  }
+
+  @include e(panel-scrollbar) {
+    height: auto;
+    // Must stay in sync with POPPER_VIEWPORT_PADDING * 2 in cascader.vue
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)